### PR TITLE
Add location info in message and contributor in README

### PR DESCRIPTION
**Pull request by turbatuandrei**
Pull request to solve issue:
https://github.com/xofbd/wqu_project_s202009/issues/1

### CODE REVIEW

**Comment by turbatuandrei** [comment]
@xofbd could you take a look at this?

File: wqu_app/message.py [hunk 1]
```diff
@@ -29,10 +30,11 @@ def get_weather(coords):
 
 
 def greet(ip_address):
-    coords = get_geolocation(ip_address)
+    coords = get_geolocation(ip_address)[0]
     temperature = get_weather(coords)
+    city = get_geolocation(ip_address)[1]
```
Comment (xofbd) [inline wqu_app/message.py]: Notice how we are calling `get_geolocation` function twice? We can combine those two calls into one.

File: wqu_app/message.py [hunk 2]
```diff
@@ -29,10 +30,11 @@ def get_weather(coords):
...
 
-    return f"Hello, the temperature is {temperature} deg C"
+    return f"Hello, the temperature in {city}(where your internet provider is located) is {temperature} deg C"
```
Comment (xofbd) [inline wqu_app/message.py]: I would have the parenthetical statement to be `(based on your IP address)`.

**Review by xofbd — CHANGES_REQUESTED** [review]
A couple of minor changes before we can merge.

**Review by xofbd — APPROVED** [review]

**Pull request merged**